defaults-jalien.sh

```diff
@@ -16,7 +16,8 @@ overrides:
   fastjet:
     tag: v3.4.0_1.045-alice1
   XRootD:
-    tag: v4.12.5
+    source: https://github.com/xrootd/xrootd
+    tag: master
```

Comment:
I do not think this is a good idea, even for just the jalien default. It will trigger a new build of ROOT every time it changes, potentially clogging our CI. If you really need to change tag that ofter, please just hardcode the commit.

Reply:
Alright, changed to use custom tag in own repo. Would that work?

Reply:
Yes.